api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/pooling/MaxPool3D.kt

```diff
@@ -0,0 +1,90 @@
+package org.jetbrains.kotlinx.dl.api.core.layer.pooling
+
+import org.jetbrains.kotlinx.dl.api.core.KGraph
+import org.jetbrains.kotlinx.dl.api.core.layer.Layer
+import org.jetbrains.kotlinx.dl.api.core.layer.convolutional.ConvPadding
+import org.jetbrains.kotlinx.dl.api.core.shape.convOutputLength
+import org.jetbrains.kotlinx.dl.api.core.shape.shapeOperand
+import org.jetbrains.kotlinx.dl.api.inference.keras.CHANNELS_FIRST
+import org.jetbrains.kotlinx.dl.api.inference.keras.CHANNELS_LAST
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import java.util.*
+
+/**
+ * Max pooling operation for 3D data (spatial or spatio-temporal).
+ * NOTE: Works with tensors which must have rank 5 (batch, depth, height, width, channels).
+ * @property [dataFormat] The ordering of the dimensions in the inputs. (CHANNEL_FIRST, CHANNEL_LAST)
+ * @property [poolSize] The size of the sliding window for each dimension of input tensor (pool batch, pool depth ,pool height, pool width, pool channels).
+ * Usually, pool batch and pool channels are equal to 1.
+ * @property [strides] Strides of the pooling operation for each dimension of input tensor.
+ * @property [padding] The padding method, either 'valid' or 'same'.
+ * @property [name] Custom layer name.
+ * @constructor Creates [MaxPool2D] object.
+ */
+public class MaxPool3D(
+    public val dataFormat: String = CHANNELS_LAST,
+    public var poolSize: IntArray = intArrayOf(1, 2, 2, 2, 1),
+    public var strides: IntArray? = null,
```

Comment:
Could we set up here the default value? Not good to have IntArray? type if it possible to avoid it